examples/html-css/index.html

```diff
@@ -3,10 +3,18 @@
   <head>
     <meta charset="utf-8" />
     <title>Design System</title>
-    <link rel="stylesheet" href="../../packages/core/dist/index.css" />
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/@cmsgov/design-system@2.0.0/dist/css/index.css"
```

Comment:
Can we leave a comment here explaining that in a real project this will lead to your local CSS file, while in this case we are using a CDN as an example